server/apiv2/handlers/tso_keyspace_group.go

```diff
@@ -347,7 +349,7 @@ func AllocNodesForKeyspaceGroup(c *gin.Context) {
 	c.JSON(http.StatusOK, nodes)
 }
 
-// SetNodesForKeyspaceGroupParams defines the params for setting nodes for keyspace groups.
+// SetNodesForKeyspaceGroupParams defines the params for setting nodes for keyspace group.
 // Notes: it should be used carefully.
 type SetNodesForKeyspaceGroupParams struct {
 	Nodes []string `json:"nodes"`
```

Comment:
Can we also support set priorities in  SetNodesForKeyspaceGroup API? if priorities []int is null, then set default priority for every node in the Nodes.
```suggestion
	Nodes     []string `json:"nodes"`
	Priorities []int `json:"priorities"`
```

Reply:
If you want to support it in the next pr, it's ok.

Reply:
now we can support "set-node <keyspace_group_id> <tso_node_addr> [<tso_node_addr>...]" and "set-priority <keyspace_group_id> <tso_node_addr> <priority>" in pd-ctl, if only run "set-node", it means to set default priority

Reply:
if we merge these interfaces, it will be not convenient in pd-ctl